plugins/stern.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: stern
+spec:
+  version: v1.20.1
+  platforms:
+  - bin: stern.exe
+    uri: https://github.com/stern/stern/releases/download/v1.20.1/stern_1.20.1_windows_amd64.tar.gz
+    sha256: 3ed6ccd050a72b68faf2f8f931969aabbd4b69fa69e1abe7ca4aaa2cf4889c22
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    files:
+    - from: stern_1.20.1_windows_amd64/*
+      to: .
+  - bin: stern
+    uri: https://github.com/stern/stern/releases/download/v1.20.1/stern_1.20.1_linux_arm64.tar.gz
+    sha256: 194bbdf77a9eaee71fa830c59db5bd9af8148d710fbe64dc656ab505ad2f66c3
+    selector:
+      matchLabels:
+        os: linux
+        arch: arm64
+    files:
+    - from: stern_1.20.1_linux_arm64/*
+      to: .
+  - bin: stern
+    uri: https://github.com/stern/stern/releases/download/v1.20.1/stern_1.20.1_linux_amd64.tar.gz
+    sha256: a7c24a6804e4c7a1fcf5ed6db1cdaa6f8ff9c4a640939c2be00c84b116668094
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    files:
+    - from: stern_1.20.1_linux_amd64/*
+      to: .
+  - bin: stern
+    uri: https://github.com/stern/stern/releases/download/v1.20.1/stern_1.20.1_darwin_amd64.tar.gz
+    sha256: ca7e2a15d0f4d848227b09041c23a333d662b2e08c6d9d48171216ea801d561c
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    files:
+    - from: stern_1.20.1_darwin_amd64/*
+      to: .
+  shortDescription: Multi pod and container log tailing for Kubernetes
```

Comment:
no need to say "for Kubernetes", it's kinda obvious since this is a kubectl plugin manager.